src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -53,7 +55,11 @@ const ShareableUrlModal = ({ onToggleModal, visible }) => {
   const [showCopied, setShowCopied] = useState(false);
   const [compatibilityData, setCompatibilityData] = useState({});
   const [canUseShareableUrls, setCanUseShareableUrls] = useState(true);
-  const [isDisclaimerViewed, setIsDisclaimerViewed] = useState(false);
+  const [showPublishedView, setShowPublishedView] = useState(false);
+  const [hostingPlatformLocalStorageVal, setHostingPlatformLocalStorageVal] =
+    useState(loadLocalStorage(localStorageSharableUrl) || {});
+  const [populatedContentKey, setPopulatedContentKey] = useState(undefined);
+  const [toggleValue, setTogleValue] = useState(false);
```

Comment:
for now it's just an empty function as i dont know what needs to go in here. It will be taken over by @jitu5 once I merge to main. 

@jitu5 do we need  isPreviewEnabled with the logic you have from your side?

Reply:
Yes, setTogleValue/isPreviewEnabled will be needed for disable preview toggle button.

Reply:
can we rename this to isPreviewEnabled, setIsPreviewEnabled ?

Reply:
yup sure 👍

Reply:
done